src/presentation/http/router/note.ts

```diff
@@ -26,10 +26,25 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
   const noteService = opts.noteService;
 
   /**
-   * Get notes
+   * Get note by id
    */
-  fastify.get('/', async () => {
-    return { hello: 'world' };
+  fastify.get<{ Params: GetNoteByIdOptions }>('/:id', async (request, reply) => {
+    const params = request.params;
+
+    const note = await noteService.getNoteById(params);
+
+    /**
+     * Check if note does not exist
+     */
+    if (!note) {
+      const payload = {
```

Comment:
API response format should be unified and described somewhere. There was a format description in Figma.

Reply:
done, added type HttpResponse